plugin/lib/class-php-compatibility-checker.php

```diff
@@ -84,7 +83,7 @@ public function init() {
 		$instance = self::instance();
 
 		// Load textdomain.
-		add_action( 'init', array( $instance, 'load_textdomain' ) );
+		// add_action( 'init', array( $instance, 'load_textdomain' ) );
```

Comment:
Commented this out because it produce PHP Warning (method does not exist)

Reply:
Removed in https://github.com/10up/phpcompat/pull/2/commits/589a2dd0d3c99f495ca4cf375330a0188c6830c1